src/chapters/aspdotnet-model-classes/model-intro.rst

```diff
@@ -0,0 +1,122 @@
+Models in MVC
+==============
+
+In the previous chapter, you learned about Razor, which displays data and an
+interface for a user. Before that, you learned about :ref:`controllers`. Remember that 
+determine what data to send to the the views. This data needs to come from some source 
```

Comment:
Is this supposed to read "Remember that controllers determine what data is sent to the views."?